src/main/notifications/index.ts

```diff
@@ -49,6 +50,10 @@ export function displayMention(title: string, body: string, channel: {id: string
         data,
     };
 
+    if (!await PermissionsManager.doPermissionRequest(webcontents.id, 'notifications', view.view.server.url.toString())) {
```

Comment:
0/5 on if this is actually an issue, but it feels weird to see an await inside a negated if condition. I'd tend to split that onto its own line

Reply:
I've been using this more and more, it's a nice shorthand. Going to keep as-is.